test_xdb/models/under_test/generate_daily_time_series_values_test.sql

```diff
@@ -1,5 +1,5 @@
 {{ config({"tags":["exclude_bigquery", "exclude_bigquery_tests"]}) }}
-SELECT 
-    {{ xdb.generate_daily_time_series_values('2020-01-01', '2020-01-05') }} AS one_day_diff
+
+SELECT {{ xdb.generate_daily_time_series_values('2020-01-01', '2020-01-05') }} AS one_day_diff
```

Comment:
This seems like a weird change without the accompanying change to the `FROM` line.